packages/containers/src/List/List.container.js

```diff
@@ -191,7 +191,12 @@ class List extends React.Component {
 		props.list.sort = {
 			field: state.sortOn,
 			isDescending: !state.sortAsc,
-			onChange: this.onSelectSortBy,
+			onChange: (event, data) => {
+				if (props.toolbar.sort.onChangeSortOrder) {
+					props.toolbar.sort.onChangeSortOrder(event, data);
```

Comment:
The props are not defined as component propTypes.

Reply:
fixed